doc/sphinx/source/examples.rst

```diff
@@ -0,0 +1,442 @@
+********
+Examples
+********
+
+This section contains a mathematical description of all examples provided with libCEED
+in the :file:`examples/` directory.
+
+For most of our examples, the spatial discretization
+uses high-order finite elements/spectral elements, namely, the high-order Lagrange
+polynomials defined over :math:`P` non-uniformly spaced nodes, the Legendre-Gauss-Lobatto (LGL)
+points (roots of the :math:`p^{th}`-order Legendre polynomial :math:`P_p`), and
+quadrature points :math:`\{q_i\}_{i=0}^Q`, with corresponding weights
+:math:`\{w_i\}_{i=0}^Q` (typically the ones given by Gauss or Gauss-Lobatto quadratures,
+that are built in the library).
+
+We discretize the domain, :math:`\Omega \subset \mathbb{R}^d` (with :math:`d=1,2,3`,
+typically) by letting :math:`\Omega = \bigcup_{e=1}^{N_e}\Omega_e`, with :math:`N_e`
+disjoint elements. For most examples we use unstructured meshes for which the elements
+are hexaedra (although this is not a requirement in libCEED).
+
+The physical coordinates are denoted by :math:`\mathbf{x}=(x,y,z)\in\Omega_e`,
+while the reference coordinates are represented as
+:math:`\boldsymbol{X}=(X,Y,Z) \equiv (X_1,X_2,X_3) \in\mathbf{I}=[-1,1]^3`
+(for :math:`d=3`).
+
+
+Standalone Ceed
+===============
+
+The following two examples have no dependencies, and are designed to be self-contained. For
+additional examples that use external discretization libraries (MFEM, PETSc, Nek5000
+etc.) see the subdirectories in :file:`examples/`.
+
+Ex1-Volume
+----------
+
+This example is located in the subdirectory :file:`examples/ceed`. It illustrates a
+simple usage of libCEED to compute the volume of a given body using a matrix-free
+application of the mass operator. Arbitrary mesh and solution orders in 1D, 2D and 3D
+are supported from the same code.
+
+This example shows how to compute line/surface/volume integrals of a 1D, 2D, or 3D
+domain :math:`\Omega` respectively, by applying the mass operator to a vector of
+:math:`\mathbf{1}`'s. It computes:
```

Comment:
Apostrophe not needed for the plural in this case

Reply:
Yes, thanks for noticing it! I had used the apostrophe just as a hack, before learning the escape character needed after the math directive. Otherwise, it was not possible just placing the `s` without an extra space. Updated now.